tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -462,27 +483,26 @@ def test_positive_validate_capsule_certificate(self):
         with get_connection(timeout=200) as connection:
             # extract the cert from the tar file
             result = connection.run(
-                'tar -xf {0}/capsule_certs.tar --directory {0}/ '.format(self.tmp_dir)
+                'tar -xf {0}/capsule_certs.tar --directory {0}/ '.format(file_setup['tmp_dir'])
             )
             assert result.return_code == 0, 'Extraction to working directory failed.'
             # Extract raw data from RPM to a file
             result = connection.run(
                 'rpm2cpio {0}/ssl-build/capsule.example.com/'
                 'capsule.example.com-qpid-router-server*.rpm'
-                '>> {0}/ssl-build/capsule.example.com/cert-raw-data'.format(self.tmp_dir)
+                '>> {0}/ssl-build/capsule.example.com/cert-raw-data'.format(file_setup['tmp_dir'])
```

Comment:
I know we are using dummy capsule hostname here, but I think it would make more sense to substitute using `.format()`
like:
```
result = connection.run(
'rpm2cpio {0}/ssl-build/{1}/'
'{1}-qpid-router-server*.rpm'
'>> {0}/ssl-build/{1}/cert-raw-data}'.format(file_setup['tmp_dir'], cert_data[capsule_hostname])
```